src/lambda/local/configureLocalLambda.ts

```diff
@@ -131,31 +132,66 @@ function getTabSize(editor?: vscode.TextEditor): number {
 
 async function loadSymbols(
     uri: vscode.Uri,
-    maxAttempts = 3,
-    retryDelayMillis = 500
-): Promise<vscode.DocumentSymbol[] | undefined> {
-    if (maxAttempts <= 0) {
-        return undefined
-    }
+    maxAttempts = 10,
+    retryDelayMillis = 200
+): Promise<vscode.DocumentSymbol[] | boolean> {
 
     const symbols = await vscode.commands.executeCommand<vscode.DocumentSymbol[]>(
         'vscode.executeDocumentSymbolProvider',
         uri
     )
+    // checking if symbols exists as this can fail if the VS Code JSON symbol provider is not yet initialized
     if (symbols) {
+        // file has symbols if JSON with at least one valid top-level key/value pair
         return symbols
     }
 
+    if (maxAttempts <= 0) {
+        // JSON symbol provider hasn't loaded and/or file exists but is unparseable (default 2 seconds total)
+        // (e.g. file is blank, only contains `{}`, or does not have any valid top level keys)
+        const localize = nls.loadMessageBundle()
```

Comment:
Move this line up just below the import statements. That's how we set up `localize` in other files